src/components/Tree/Tree.spec.tsx

```diff
@@ -8,21 +8,23 @@ import { mockNodesFlat } from '@components/Tree/utils';
 
 type ComponentProps = {
     nodes: DraggableItem<TreeFlatListItem>[];
-    onDrop?: (modifiedItems: DraggableItem<TreeFlatListItem>[]) => void;
+    onDrop?: (sourceItemId: string, parentId: NullableString, positionBeforeId: NullableString) => void;
     onEditableSave?: (itemId: string, value: string) => void;
 };
 const Component: FC<ComponentProps> = ({ nodes, onDrop, onEditableSave }) => {
     const [selectedIds, setSelectedIds] = useState<NullableString[]>([]);
-    const onDropDefault = (items: DraggableItem<TreeFlatListItem>[]) => {
-        console.log(items);
+    const onDropDefault = (sourceItemId: string, parentId: NullableString, positionBeforeId: NullableString) => {
+        console.log(sourceItemId);
+        console.log(parentId);
+        console.log(positionBeforeId);
     };
 
     return (
         <Tree
             nodes={nodes}
             activeNodeIds={selectedIds}
             onSelect={(ids: NullableString[]) => setSelectedIds(ids)}
-            onUpdate={onDrop || onDropDefault}
+            onSortUpdate={onDrop || onDropDefault}
```

Comment:
maybe change the name from onSortUpdate to onDragAndDrop or onSort?

Reply:
Renamed to onDragAndDrop